src/core/pool_type.hpp

```diff
@@ -375,7 +374,7 @@ struct Pool : PoolBase {
 		 * @note when this function is called, PoolItem::Get(index) == nullptr.
 		 * @note it's called only when !CleaningPool()
 		 */
-		static inline void PostDestructor(size_t index) { }
+		static inline void PostDestructor([[maybe_unused]] size_t index) { }
```

Comment:
Why `[[maybe_unused]]` over removing the parameter name?

Reply:
This is a method in a base-class. The parameter is required for documentation, including the "@param".